ci/nightly/pipeline.template.yml

```diff
@@ -38,19 +38,40 @@ steps:
     agents:
       queue: builder-linux-x86_64
 
-  - id: feature-benchmark
-    label: "Feature benchmark against merge base or 'latest'"
-    timeout_in_minutes: 360
-    agents:
-      queue: linux-x86_64-large
-    artifact_paths: junit_*.xml
-    plugins:
-      - ./ci/plugins/mzcompose:
-          composition: feature-benchmark
-          args:
-             - --other-tag
-             # common-ancestor will default to latest if not in a PR
-             - common-ancestor
+  - group: Benchmarks
+    key: benchmark
+    steps:
+    - id: feature-benchmark
+      label: "Feature benchmark against merge base or 'latest'"
+      timeout_in_minutes: 360
+      agents:
+        queue: linux-x86_64-large
+      artifact_paths: junit_*.xml
+      plugins:
+        - ./ci/plugins/mzcompose:
+            composition: feature-benchmark
+            args:
+               - --other-tag
+               # common-ancestor will default to latest if not in a PR
+               - common-ancestor
+    - id: scalability-benchmark
+      label: "Scalability benchmark against merge base or 'latest'"
+      timeout_in_minutes: 90
+      agents:
+        queue: linux-x86_64-large
+      artifact_paths: junit_*.xml
+      plugins:
+        - ./ci/plugins/mzcompose:
+            composition: scalability
+            args:
+              - --target
+              - HEAD
+              - --target
+              - common-ancestor
+              - --regression-against
+              - common-ancestor
+              - --max-concurrency
```

Comment:
I think 256 is the default already in mzcompose?

Reply:
Yes, but I don't think it hurts to still have it specified here.